addons/mail/models/models.py

```diff
@@ -123,6 +123,9 @@ def _notify_get_reply_to(self, default=None, records=None, company=None, doc_nam
                 if not doc_names:
                     doc_names = dict((rec.id, rec.display_name) for rec in _records)
 
+                if not company and 'company_id' in self and len(self.company_id) == 1:
```

Comment:
This is a simplification that does not hold in any MC process ... seems strange to make a fix that will be broken in that case.

Reply:
if a record has no company or multiple ones we keep the current behaviour (default the user company)
this fix will address the case when a record  is linked to a single company

Reply:
Well for 14 that's ok I suppose